wiki/Glossary/en.md

```diff
@@ -34,65 +34,62 @@ Authentic is a word that denotes a beatmap that emulates in every aspect of a sp
 ### Ban
 
 A ban is an action taken against a player if caught violating a rule (i.e. multi-accounts or cheating).
-However, there is a step-by-step process that users follow before getting banned, in most cases.
+However, there is a step-by-step process that users follow before getting banned in most cases.
```

Comment:
None of this section is very relevant anymore. We don't refer to "bans" outside of /p/bans, they're called "restrictions" and this entire sentence doesn't make any sense whatsoever. Might be beyond the scope of this commit to adjust this here, though.